adr/ADR-51.md

```diff
@@ -53,14 +53,14 @@ There may only be one message per subject that holds a schedule, if a user wishe
 In this use case the Stream holds a message with a Cron-like schedule attached to it and the Stream will produce messages on the given schedule.
 
 ```bash
-$ nats pub -J '$SCHED.update_orders' \
+$ nats pub -J 'orders_schedules' \
```

Comment:
Maybe use `orders.schedule` as the schedule and `orders.delayed` as target?
The `$SCHED` was used so it could be captured by a stream with `$SCHED.>` as subject. So changing it to the above would allow to infer using `orders.*` or `orders.>` as the stream subject.